tests/test_external.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from nose.tools import assert_true, assert_equal
+from datajoint.external import ExternalTable
+
+from . schema_external import schema
+
+
+def test_external_put():
```

Comment:
need to add test case for pure `external` in addition to `external-xxx` formatted configs